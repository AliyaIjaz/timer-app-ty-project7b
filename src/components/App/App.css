.app-container {
 height: 85vh;
 width: 85vw;
 align-items: center;
 display: flex;
 justify-content: center;
 margin: auto;
}