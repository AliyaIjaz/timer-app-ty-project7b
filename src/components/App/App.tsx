import './App.css';
import Timer from '../Timer/Timer'
import Header from '../Header/Header'

function App() {
  return (
    <div>
      <Header />
      <div className='app-container'>
        <Timer />
      </div>
    </div>
  )
}
export default App;
