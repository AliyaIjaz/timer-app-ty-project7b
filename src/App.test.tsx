import { shallow } from 'enzyme';
import App from './components/App/App';

 
test('renders the component', () => {
  const component = shallow(<App />);
  expect(component.find('div').length).toEqual(2);
});