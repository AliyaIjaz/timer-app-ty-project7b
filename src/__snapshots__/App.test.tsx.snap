// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div>
<Header />
<div
  className="app-container"
>
  <Timer />
</div>
</div>
`;
